cmd/notificationsemail.go

```diff
@@ -376,6 +395,7 @@ var updateEmailNotificationCmd = &cobra.Command{
 func init() {
 	addNotificationEmailCmd.Flags().StringP("name", "n", "", "The name of the notification")
 	addNotificationEmailCmd.Flags().StringP("email", "E", "", "The email address of the notification")
+	addNotificationEmailCmd.Flags().Uint("organizationID", 0, "ID of the Organization")
```

Comment:
In the interest of standardising. New flags should use kebab-case.  `organizationID` should become `organization-id`.

I know there are some flags in older commands that use camelCase, but we should phase these out over time in existing commands.